react/src/js/react/discover/PostsStore.js

```diff
@@ -21,6 +22,9 @@ export class PostsStore extends Reflux.Store {
             creatingPost: false,
             successfullyCreatedPost: false,
             failedToCreatePost: false,
+            updatingPost: false,
```

Comment:
For clarity, this should be "creatingComment"
The other state attributes you added should have Comment in their name, as opposed to Post